NuGet.config

```diff
@@ -25,8 +25,8 @@
     <add key="vs-devcore" value="https://myget.org/F/vs-devcore/api/v3/index.json" />
     <add key="vs-editor" value="https://myget.org/F/vs-editor/api/v3/index.json" />
     <add key="roslyn_concord" value="https://myget.org/F/roslyn_concord/api/v3/index.json" />
-    <add key="vssdk" value="https://vside.myget.org/F/vssdk/api/v3/index.json" />
-    <add key="vs-impl" value="https://vside.myget.org/F/vs-impl/api/v3/index.json" />
+    <add key="vssdk" value="https://pkgs.dev.azure.com/azure-public/vside/_packaging/vssdk/nuget/v3/index.json" />
+    <add key="vs-impl" value="https://pkgs.dev.azure.com/azure-public/vside/_packaging/vs-impl/nuget/v3/index.json" />
```

Comment:
@mavasani @RikkiGibson I didn't make these changes.  However, I also don't know what to do about this.  I think i need these changes to go in so that that feature branch you built can go through.  But it seems very strange that this needs to be done, and i'm not sure how we possibly review that these changes are appropriate.